docs/developers/modules/marketmap.mdx

```diff
@@ -0,0 +1,292 @@
+---
+description: Reference documentation for the Marketmap module.
+title: Marketmap
+icon: message-dollar
+---
+
+## Overview
+
+The `Marketmap` module is responsible for managing and storing a configuration that informs the Connect oracle of which markets to fetch data for, and which providers to use to fetch them.
+
+The `Marketmap` module's Keeper is instantiated with an `Authority` address, guarding the mutation of the `Marketmap` by a specific address. This address can be the `governance` address, allowing governance propositions to mutate the `Marketmap`.
```

Comment:
```suggestion
The `Marketmap` module's Keeper is instantiated with an `Admin` address, guarding the mutation of the `Marketmap` by a specific address. This address can be the `governance` address, allowing governance propositions to mutate the `Marketmap`.
```

Reply:
we seem to use admin/authority interchangeably, the reason i opted for "Authority" here is because of the argument name in the NewKeeper constructor

with that said, do you feel it should still be Admin?

Reply:
Ok so there's 3 things:
- `Authority` (generally gov address which can change params)
- `Admin` (address appointed by gov that can remove `MarketAuthorities`)
- `MarketAuthorities` ( addresses that are authorized to update the `marketmap`)

Reply:
Admin and MarketAuthorities could be gov or just any general address, and can act instantly rather than having to wait like in governance

Reply:
updated [more docs on the authority, admin, and market authority](https://github.com/skip-mev/slinky/pull/630/commits/295255354ec912a35e4eeed4ec295f3a6bc53e40)